gptme/chat.py

```diff
@@ -235,9 +235,6 @@ def step(
         if msg_response:
             yield msg_response.replace(quiet=True)
             yield from execute_msg(msg_response, confirm)
-    except KeyboardInterrupt:
```

Comment:
This is now handled in the `execute_msg` as we need the `call_id` in the message response.